### PR TITLE
[v6r16]  Fix setupSite

### PR DESCRIPTION
**Pull request by atsareg**
Refresh database configuration even if the database exists in the server
Refresh global configuration when adding options to the local dirac.cfg

### CODE REVIEW

File: Core/scripts/dirac-setup-site.py [hunk 1]
```diff
@@ -7,15 +7,30 @@
...
+
+from DIRAC import S_OK
 from DIRAC.Core.Base import Script
+
+class Params:
+
+  def __init__( self ):
+    self.exitOnError = False
+
+  def setExitOnError( self, value ):
```
Comment (fstagni) [inline Core/scripts/dirac-setup-site.py]: What's `value` for?

File: FrameworkSystem/Client/ComponentInstaller.py [hunk 2]
```diff
@@ -287,6 +287,11 @@ def _addCfgToDiracCfg( self, cfg, verbose = False ):
     if not result:
       return result
     self.loadDiracCfg( verbose )
+
+    # Make sure that new options are globally available
+    if gConfig:
```
Comment (fstagni) [inline FrameworkSystem/Client/ComponentInstaller.py]: is this `if` necessary?

File: Core/scripts/dirac-setup-site.py (hunk 1 above)
Comment (atsareg) [inline Core/scripts/dirac-setup-site.py]: This is a standard callback signature

File: FrameworkSystem/Client/ComponentInstaller.py (hunk 2 above)
Comment (atsareg) [inline FrameworkSystem/Client/ComponentInstaller.py]: Not really I think, the problem was actually elsewhere, I will scrap it

Comment (atsareg) [inline FrameworkSystem/Client/ComponentInstaller.py]: Removed, please continue review

**Pull request merged**